Automattic/vip-cli
# Add/local dev env

## Description

Adds the first step towards integrating local dev environment into `vip` command.

This adds `vip dev-environent start` command. Which will create (if non-existing) and start the environment.

It is a slightly reduced version of our internal tool. Things I dropped to make it easier for the first step (we can add them later):
- lookup of config by site id
- auto cloning of clients code (client can still clone their code manually and point to the path of that)  
- more operations (stop/destroy) - TBA in follow-up work

Maybe the command should be first hidden behind a isVIP flag for the first iteration untill we get support for all operations. For that we could use https://github.com/Automattic/vip/pull/699 after merged. 

## Steps to Test

Try variants of command. Use `--help` to learn about a possible parameter. Please let me know if help doesn't make sense or is incomplete in some sense.

It's usefull to have `export DEBUG="@automattic/vip:bin:dev-environment,@automattic/vip:bin:vip"` to see more info like instance path so you can remove that folder to start fresh.
